TechliftTelegramBot/Services/GenerateKeyboard.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using TechliftTelegramBot.Models;
+using Telegram.Bot.Types.ReplyMarkups;
+
+namespace TechliftTelegramBot.Services
+{
+    public class GenerateKeyboard
```

Comment:
have the class implement an interface and pass it as a service wherever this is needed